pkg/hintrunner/zero/zerohint_ec_test.go

```diff
@@ -688,5 +688,50 @@ func TestZeroHintEc(t *testing.T) {
 				}),
 			},
 		},
-	})
+		"ReduceV1": {
+			{
+				operanders: []*hintOperander{
+					{Name: "x.d0", Kind: apRelative, Value: &utils.FeltZero},
+					{Name: "x.d1", Kind: apRelative, Value: &utils.FeltZero},
+					{Name: "x.d2", Kind: apRelative, Value: &utils.FeltZero},
+				},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					hinter.InitializeScopeManager(ctx, map[string]any{})
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newReduceV1Hint(ctx.operanders["x.d0"])
+				},
+				check: varValueInScopeEquals("value", bigIntString("0", 10)),
+			},
+			{
+				operanders: []*hintOperander{
+					{Name: "x.d0", Kind: apRelative, Value: feltString("3618502788666131213697322783095070105623107215331596699973092056135872020482")},
+					{Name: "x.d1", Kind: apRelative, Value: feltString("0")},
```

Comment:
you can use &utils.FeltZero here and in other places where you use felt with value of 0